### ISSUE TITLE
World Food Programe Comms

### ISSUE THREAD

**Issue by Cotabe** [Giveth/giveth-planning#1128]
Context:

The Global Blockchain Business Council Giving (GBBC Giving) and the World Food Program (WFP) have partnered to ignite a 1 billion USD project to address the hunger crisis.

We had a couple of calls and they are very interested to have Giveth as a partner in this initiative, especially providing input and technology on donations acceptance, global transfer, traceability and supporting local innovators.

The exact mechanisms of collaboration have not yet been established. But we will probably be collaborating in working groups.

The project aims to be ideated and ready to pilot by Oct. 2023 and ready to scale by Sep. 2024. So a lot of things to be figured out yet. However, this program may be officially announced on Davos Jan 16-20 and they plan to reveal some of their partners, Giveth included. 

Therefore, I consider it's in our best interest to have comms around it. Currently, we are blocked on the specifics of the announcement as most people on the WFP and the GBBC Giving will be on vacation till next week.

I think the most qualified people to get into the specifics on what's the best way to go with these comms are @laurenluz and @c0ric0ri 

IMO, since it's only going to be an announcement of partners in a project about to kick off. Maybe it's best to keep it simple with a tweet sharing the announcement, the main goals of the projects and why we are so excited to be part of it. But I suggest we are just prepared an conscious this is coming and wait till next week to get more info from the WFP and the GBBC Giving.

Here is a link with slides about the initiative https://drive.google.com/file/d/1c8LtGVZXs_EhBHQSuTCmNhGhX6-kH83S/view?usp=sharing
I won't make it public as I'm not sure they are public. But feel free to request access.

**Comment by laurenluz** [comment]
@c0ric0ri can you draft something for this?

also wondering why you set the date to Jan 13? if they are only maybe going to announce Giveth as a partner and won't do so until Jan 16, I'm wondering what we'd say before seeing the tweet.

Seems like the best way is to quote tweet their announcement w/ a bit more detail. Unless you have another idea, I'm open.

**Comment by c0ric0ri** [comment]
@laurenluz because with our points of contact coming back the week of the 9th, I would aim to have something drafted and reviewed by the end of the week on the 13th. 

I'm assuming they will provide us with information next week to set up a quote tweet or whatever is most appropriate, but we should have it ready to post for Monday the 16th.
